### PR TITLE
Correct RegisterCloudProvider link

### PR DESCRIPTION
**Pull request by RainbowMango**
Minor change:

Correct `func RegisterCloudProvider(name string, cloud Factory)` link.
https://github.com/kubernetes/cloud-provider/blob/master/plugins.go#L55-L63

### CODE REVIEW

**Comment by RainbowMango** [comment]
/assign @tengqm 
as @xiangpengzhao GitHub status is `no enough time on github recently` .

File: content/en/docs/tasks/administer-cluster/developing-cloud-controller-manager.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ can develop their features independently from the core Kubernetes release cycles
...
 
-To dive a little deeper into implementation details, all cloud controller managers will import packages from Kubernetes core, the only difference being each project will register their own cloud providers by calling [cloudprovider.RegisterCloudProvider](https://github.com/kubernetes/cloud-provider/blob/master/plugins.go#L56-L66) where a global variable of available cloud providers is updated.
+To dive a little deeper into implementation details, all cloud controller managers will import packages from Kubernetes core, the only difference being each project will register their own cloud providers by calling [cloudprovider.RegisterCloudProvider](https://github.com/kubernetes/cloud-provider/blob/master/plugins.go#L55-L63) where a global variable of available cloud providers is updated.
```
Comment (tengqm) [inline content/en/docs/tasks/administer-cluster/developing-cloud-controller-manager.md]: pls pin to a specific branch or tag. do not use master which is a moving target.

Comment (RainbowMango) [reply content/en/docs/tasks/administer-cluster/developing-cloud-controller-manager.md]: It makes sense.

Comment (RainbowMango) [reply content/en/docs/tasks/administer-cluster/developing-cloud-controller-manager.md]: Done. PTAL again.

**Pull request merged**